docs/eventing/broker/kafka-broker.md

```diff
@@ -0,0 +1,112 @@
+[Kafka Broker](https://github.com/knative-sandbox/eventing-kafka-broker).
+
+This guide assumes a 
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component).
+
+## Installation
+
+<!-- TODO Where does the config for populating the following live? -->
+```bash
+kubectl apply --filename {{< artifact repo="eventing-kafka-broker" file="eventing-kafka-broker.yaml" >}}
+```
+## Create a Kafka Broker
+
+A Kafka Broker object looks like this:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    # case-sensitive
+    eventing.knative.dev/broker.class: kafka 
+  name: default
+  namespace: default
+spec:
+  # Configuration specific to this broker.
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: kafka-broker-config
+    namespace: knative-eventing
+  # Optional dead letter sink, you can specify either:
+  #  - deadLetterSink.ref, which is a reference to a Callable
+  #  - deadLetterSink.uri, which is an absolute URI to a Callable (It can potentially be out of the Kubernetes cluster)
+  delivery: 
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: dlq-service
+```
+
+`spec.config` should reference any `ConfigMap` that looks like the following:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kafka-broker-config
+  namespace: knative-eventing
+data:
+  # Number of topic partitions
+  default.topic.partitions: "10"
+  # Replication factor of topic messages.
+  default.topic.replication.factor: "1" 
+  # A comma separated list of bootstrap servers. (It can be in or out the k8s cluster)
+  bootstrap.servers: "my-cluster-kafka-bootstrap.kafka:9092" 
+```
+
+The above `ConfigMap` is installed in the cluster, and you can edit
+the configurations or create a new one with the same shape 
+depending on your needs.
+
+## Set as default broker implementation
+
+To set the Kafka Broker as your default implementation you can edit
+the `config-br-defaults` `ConfigMap` in `knative-eventing` namespace.
+In this way, you can avoid specifying 
+`metadata.annotations.eventing.knative.dev/broker.class` or 
+`spec.config` for each Kafka Broker created in the cluster
+or in a specific namespace.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-br-defaults
+  namespace: knative-eventing
+data:
+  default-br-config: |
+    clusterDefault:
+      brokerClass: kafka 
+      apiVersion: v1
+      kind: ConfigMap
+      name: kafka-broker-config
+      namespace: knative-eventing
+    namespaceDefaults:
+      namespace1:
+        brokerClass: kafka 
+        apiVersion: v1
+        kind: ConfigMap
+        name: kafka-broker-config
+        namespace: knative-eventing
+      namespace2:
+        brokerClass: kafka 
+        apiVersion: v1
+        kind: ConfigMap
+        name: kafka-broker-config
+        namespace: knative-eventing
+```
+
```

Comment:
Should you spend a bunch of more words about the other CMs (logging, kafka properties, etc?)

Reply:
Yeah, let me do that

Reply:
Done: https://github.com/knative/docs/pull/2740/commits/cb6dc488fc3f7d7e239aec72954adf040fda4676